_includes/install/trouble/rc_php-version.md

```diff
@@ -32,6 +32,9 @@ To resolve the issue, try the following:
 *	Use a [`phpinfo.php`]({{ page.baseurl }}/install-gde/prereq/optional.html#install-optional-phpinfo) page to collect more information
 *	Make sure you're running a supported PHP version according to our System Requirements:
 
-	*	[Version 2.0]({{ page.baseurl }}/install-gde/system-requirements.html)
+	*	[Version 2.0]({{ page.gdeurl }}/install-gde/system-requirements.html)

```

Comment:
@hiteshvaghasiya83 
Please remove forward slash form below link.

**Old** 
> [Version 2.0]({{ page.gdeurl }}/install-gde/system-requirements.html)

**Upadated** 
> [Version 2.0]({{ page.gdeurl }}install-gde/system-requirements.html)

Reply:
@atulksigma 
Thanks for the suggestion. 
Suggested change is updated on rc_php-version.md file.